lib/paperclip-azure.rb

```diff
@@ -1,13 +1,43 @@
-require 'azure'
+require 'openssl'
 
 require File.join(File.dirname(__FILE__), 'paperclip', 'storage', 'azure')
 
 module Azure
-  module Blob
-    BlobService.class_eval do
-      def initialize(signer=Core::Auth::SharedKey.new, account_name=Azure.config.storage_account_name)
-        super(signer, account_name)
-        @host = Paperclip::Storage::AzureRegion.url_for account_name
+  module Core
```

Comment:
Are these changes required if i need to remove azure and bump azure-storage-blob to v2.0.1. I have forked this repo a while ago and made quite a lot of changes to it. I am now in the process of removing azure but getting the error `cannot load such file -- azure (LoadError)` while trying to load rails console. 
Checked ChangeLog of azure-storage-blob gem and i don't see any breaking changes regarding this ( ref: [https://github.com/Azure/azure-storage-ruby/blob/v2.0.1-blob/blob/ChangeLog.md](https://github.com/Azure/azure-storage-ruby/blob/v2.0.1-blob/blob/ChangeLog.md) ). So do i need to make these changes to my forked repo as well?